source/help/policies/content-types.md

```diff
@@ -87,6 +87,10 @@ Requirements for Comments and Replies:
 - "Comments-on" content type must have a Title: "Comment(s) on '<title-of-original-paper>'" and the Comments field must contain "comment(s) on <arXiv-ID>".
 - A response from the original authors is always permitted and must have a Title "Reply to comment(s) on '<title-of-original-paper>'" and the Comments field must contain "reply to comment(s) on <arXiv-ID>" in the comment field.
 - In rare cases where the original paper is not on arXiv (see content rules) the arXiv-ID is replaced by a DOI.
-- A further chain of responses and counter-responses is allowed as version updates of the original two above; these will not be accepted as new papers with new arXiv-IDs. A given author or co-author may submit only one Comment on a particular paper. Any subsequent issues the author wishes to raise must be done through version updates of the original Comment. 
+- A further chain of responses and counter-responses is allowed as version updates of the original two above; these will not be accepted as new papers with new arXiv-IDs. A given author or co-author may submit only one Comment on a particular paper. Any subsequent issues the author wishes to raise must be done through version updates of the original Comment.
+
+  * Using a replacement to create a new version of under the same arXivID could either be done by:
```

Comment:
This doesn't need to be a bullet point. 

Is it "arXiv-ID" or "arXivID"?

Suggest this read, "A replacement can be used two ways to create a new version under the same arXiv-ID:"